input/system/google_cloudsql/logs.go

```diff
@@ -203,14 +206,12 @@ func setupLogTransformer(ctx context.Context, wg *sync.WaitGroup, servers []*sta
 					continue
 				}
 
-				parser := server.GetLogParser()
-
 				// We ignore failures here since we want the per-backend stitching logic
 				// that runs later on (and any other parsing errors will just be ignored).
 				// Note that we need to restore the original trailing newlines since
 				// AnalyzeStreamInGroups expects them and they are not present in the GCP
 				// log stream.
-				logLine, _ := parser.ParseLine(in.Content + "\n")
+				logLine, _ := server.GetLogParser().ParseLine(in.Content + "\n")
```

Comment:
Since `in` is coming from a channel and we check `ok`, I think it's impossible for `in` to be nil. But even if it was, lines of code above this would have already panicked by accessing `in`.